.github/workflows/pr-check.yaml

```diff
@@ -24,6 +24,8 @@ jobs:
     name: Windows
     runs-on: windows-2022
     timeout-minutes: 60
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
```

Comment:
it's not secrets.something ?

Reply:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#example-1-passing-the-github_token-as-an-input 

```
${{ secrets.GITHUB_TOKEN }}
```

Reply:
It is also in context https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
But I change to `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` to unify the usage